packages/plugin-conversation/src/to-array.js

```diff
@@ -0,0 +1,23 @@
+import {isFunction} from 'lodash';
+
+/**
+ * Helper to convert objects into arrays of transforms. probably belongs in
+ * spark-core
+ * @param {string} direction
```

Comment:
This should probably be marked `@private` because it's mostly just a helper to simplify the high number of transforms provided by this plugin.